cylc/flow/network/schema.py

```diff
@@ -2180,20 +2180,28 @@ class Arguments(TaskMutation.Arguments, FlowMutationArguments):
 class Trigger(Mutation, TaskMutation):
     class Meta:
         description = sstrip('''
-            Manually trigger tasks.
+            Manually trigger tasks, even in a paused workflow.
 
-            Warning: waiting tasks that are queue-limited will be queued if
-            triggered, to submit as normal when released by the queue; queued
-            tasks will submit immediately if triggered, even if that violates
-            the queue limit (so you may need to trigger a queue-limited task
-            twice to get it to submit immediately).
+            Triggering a task that is not yet queued will queue it.
+               
+            Triggering a queued task runs it immediately.
+                        
+            Queues release tasks to run when their active task count
+            drops below the queue limit. So, depending on the task count, you
+            may need to trigger a task twice to make it run immediately. 
```

Comment:
> when their active task count drops below the queue limit

Accurate, but maybe not the clearest for users who may not know what Cylc queues are.

Suggestion:
* Added an explanation of what queues are and referred to them as "Cylc" queues to differentiate from job runner queues.
* Updated "depending on the task count" to "on the **active** task count".

```suggestion
            Cylc queues restrict the number of jobs that can submit
            / run simultaneously.
            Queues release tasks to run when their active task count
            drops below the queue limit. So, depending on the active task count, you
            may need to trigger a task twice to make it run immediately. 
```

Reply:
I like that, will accept and copy...